<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teachers_2</class>
 <widget class="QMainWindow" name="teachers_2">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1572</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teachers</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="teachers_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>1551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Назад в главное меню</string>
    </property>
   </widget>
   <widget class="QLabel" name="teachers">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
</string>
    </property>
    <property name="text">
     <string>Преподаватели</string>
    </property>
   </widget>
   <widget class="QLabel" name="Idd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="familiya">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>Фамилия:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_tea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>1551</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: rgba(255,255,255,10);
    border: 1px solid rgba(255,255,255,10);
    border-raduis: 10px;
    color: white;
    font-weight: bold;
    font-size:15pt;
}
QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1.25, stop:0 #5C3566, stop: 0.6 #623A6F, stop:1 #663F79);
    color: #e1e5cf;

    border-right: 2px solid #000000; 
    border-top: 2px solid #000000; 
    border-left: 2px solid #000000; 

    font-family: Roboto;
    font-style: Regular;
    font-weight: bold;
    font-size: 10pt;

	color: white;
}

 

</string>
    </property>
    <column>
     <property name="text">
      <string>ИД</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Почта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ИД-предмета</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Search_btn_tea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Поиск по записи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Insert_tea">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>100</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Записать в БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clears_tea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>690</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Очистить ввод</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_tea">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>160</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Обновить запись в БД</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_seatch_tea">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>690</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}

</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="deletezapic_tea">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>220</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}

QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Удалить запись из БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vuvod_tea">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>690</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Вывод данных</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2_tea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3_tea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4_tea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cleant_tea">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>690</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-raduis: 7px;
color: white;
font-weight: bold;
font-size:20pt;
}
QPushButton:hover {
background-color: rgba(0, 255, 120, 0.4);
}
QPushButton:pressed {
background-color: rgba(255,255,255,70);
}
</string>
    </property>
    <property name="text">
     <string>Очистить таблицу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="otchestvo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>Отчество:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_sub">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>ИД_предмета:</string>
    </property>
   </widget>
   <widget class="QLabel" name="number">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5_tea">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6_tea">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="E_mail">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135,255), stop:0.427447 rgba(41, 61, 132,235), stop:1 rgba(155,79,165,255));
border-style: solid;
border-radius:21px;
color: white;
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
font-size:15pt;</string>
    </property>
    <property name="text">
     <string>Почта:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8_tea">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgba(255,255,255,50);
border: 1px solid rgba(255,255,255,40);
border-raduis: 10px;
color: white;
font-weight: bold;
font-size:20pt;
}
QLineEdit:hover {
background-color: rgba(255,255,255, 0.5);
}
</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
